cvat/apps/engine/media_extractors.py

```diff
@@ -752,6 +753,12 @@ def _add_video_stream(self, container, w, h, rate, options):
         if w % 2:
             w += 1
 
+        # x264 has 4K limitations,https://github.com/opencv/cvat/issues/7425
```

Comment:
Looks like only libopenh264 has a 4K limitation, but libx264 (which is used in development mode) does not.
However, it is OK to raise an exception without checking which codec is used to unify behavior. 
```suggestion
        # libopenh264 has 4K limitations, https://github.com/opencv/cvat/issues/7425
```